resources/templates/mustache/preprod/govuk-template.mustache.html

```diff
@@ -241,7 +241,18 @@
                 <ul class="platform-help-links">
                     <li><a href='/help/cookies' target="_blank" data-sso="false" data-journey-click="footer:Click:Cookies">Cookies</a></li>
                     <li><a href='/help/privacy' target="_blank" data-sso="false" data-journey-click="footer:Click:Privacy policy">Privacy policy</a></li>
+
+                    {{#termsAndConditionFooterLinks}}
```

Comment:
Do you intend for there to be multiple terms and conditions links? IF not could you rename this to  {{#termsAndConditionsFooterLink}} Plural Conditions and Singular Links. If you do intend to supply multiple t&c links could you just pluralise Conditions.

Reply:
@gordonmcmullan : Yeah its single link. I will rename it.